examples/Lombok/src/main/java/com/dslplatform/lombok/Example.java

```diff
@@ -14,6 +14,7 @@
 	public static class Employee {
 		private String name;
 		private int id;
+		private boolean active;

```

Comment:
Can you please just comment this out, as examples require a new build.
Instead add such relevant test (with boolean isSomething) to tests-java8 project

Reply:
- [ ] Add test :)

Sorry to skip it in the first place, I kinda wanted your input, as I was a bit confused on which module to put it (the luck of the first timer). Let me know if there is something else to add.

Best regards.

Reply:
Np. Tnx for the PR.

The only potentially remaining issue is the beanOrActualName method.
It might be prudent to make it more strict by passing in type argument and allowing is only for booleans, but it does not seem that big of an issue.

I'll merge this in later today. If you want to explore this change do it in another PR.
Also, this would be reflection relevant (which is much lower priority), so it should be tested in java8 project